src/checker.ts

```diff
@@ -138,7 +138,12 @@ export class LicenseChecker extends EventEmitter {
     // https://docs.npmjs.com/files/package.json#license for details. The code
     // below is a little complicated to deal with those cases.
     const license = pkgJson.license || pkgJson.licenses;
-    if (!license) return null;
+    if (!license) {
+      if (pkgJson.private) {
```

Comment:
Wouldn't it make sense for us to require private package to have a valid license field? I am concerned about a case where I may legitimately using a private package from elsewhere that changes its license at some point in time to no longer be green in the context of the dependent project.

Would it be unreasonable to ask that monorepos still have a value in `pkgJson.license` rather than hard-coding the string `'private'`?

Reply:
I agree that a private package should have license. npmjs spec even suggests to have a private license, like `{ "license": "UNLICENSED" }` for such packages.

But what I do here is an additional check if it meets a situation when a package.json doesn't have license, at this case for a private we can introduce it as a license, which is easier to catch/accept/log/etc. Basically a workaround for an existing packaged which just followed standard lerna workspace tutorials. 
If a private package has a license specified, then it will work as usual.

Reply:
👍